python/Ganga/Runtime/bootstrap.py

```diff
@@ -181,6 +181,10 @@ def manualExportToGPI(my_interface=None):
     from Ganga.GPIDev.Lib.Registry.JobRegistry import jobSlice
     exportToInterface(my_interface, "jobSlice", jobSlice, "Functions")
 
+    """Needed for the JobTime if we plan to use the interface to eval the XML as well"""
+    import datetime
+    exportToInterface(my_interface, "datetime", datetime, 'Classes')
```

Comment:
Mostly because it feels like code smell that we're having to explicitly export a particular standard module into our GPI namespace so that another part of code can continue to work. `bootstrap.py` should not need to know about `JobTime`. What if some third-party plugin requires some other module to be present, would we add them all here? When you're working on a coding problem and you're finding that you're having to add strange hacks to make things work, that's very often a sign that there's something wrong. This is coupling unrelated parts of the software together, reducing encapsulation and destroying locality when reading the code. Ganga is built on these hacks and I've been working hard to make things maintainable in the future, the only way Ganga will survive with the amount of manpower we have is if it becomes simpler and smaller. Constantly patching problems on a half-a-million LOC project does not scale.

Python is a good enough language to allow us to act polymorphicly, at the point we need to and when we need to rather than maintaining a large and complicated global state which reduces clarity and makes testing harder.